### PR TITLE
Two minor docs improvements.

### PR DESCRIPTION
**Pull request by beorn7**
@juliusv @fabxc

### CODE REVIEW

File: content/docs/practices/naming.md [hunk 1]
```diff
@@ -6,24 +6,24 @@ sort_rank: 1
...
-A metric name:
+A metric name...
 
-* should have a (single-word) application prefix relevant to the containing Prometheus domain
- * <code><b>prometheus</b>\_notifications\_total</code>
- * <code><b>indexer</b>\_requests\_latencies\_milliseconds</code>
- * <code><b>processor</b>\_requests\_total</code>
-* must have a single unit (i.e. do not mix seconds with milliseconds)
-* should have a units suffix
+* ...should have a (single-word) application prefix relevant to the domain the metric belongs to. The prefix is sometimes referred to as `namespace` by client libraries. For metrics specific to an application, the prefix is usually the application name itself. Sometimes, however, metrics are more generic, like standardized metrics exported by client libraries. Examples:
```
Comment (juliusv) [inline content/docs/practices/naming.md]: I think you should be able to wrap this text, even if it's in a list (indent everything by two spaces?)?

File: content/docs/practices/naming.md [hunk 2]
```diff
@@ -6,24 +6,24 @@ sort_rank: 1
...
+ * <code><b>prometheus</b>\_notifications\_total</code> (specific to the Prometheus server)
+ * <code><b>process</b>\_cpu\_seconds\_total</code> (exported by many client libraries)
+ * <code><b>http</b>\_request\_duration\_microseconds</code> (for all HTTP requests)
+* ...must have a single unit (i.e. do not mix seconds with milliseconds).
+* ...should have a suffix describing the unit.
```
Comment (juliusv) [inline content/docs/practices/naming.md]: `_total` is not a really a unit. To make it even more confusing, we have metrics with just `_total` and others with `<unit>_total` (like `foobar_seconds_total`). Maybe add an extra comment that counters should have an additional `_total` suffix *after* the unit?

File: content/docs/practices/naming.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ Use labels to differentiate the characteristics of the thing that is being measu
 Do not put the label names in the metric name, as this introduces redundancy
 and will cause confusion if the respective labels are aggregated away.
 
-CAUTION: <b>CAUTION:</b> Remember that every unique key-value label pair
+**CAUTION:** Remember that every unique key-value label pair
```
Comment (juliusv) [inline content/docs/practices/naming.md]: The initial `CAUTION` here triggers a special "caution" markup block and doesn't actually show up in the text. See http://prometheus.io/docs/practices/naming/. It'll need to be preserved.

File: content/docs/practices/naming.md (hunk 2 above)
Comment (beorn7) [inline content/docs/practices/naming.md]: This section is not about units. The _total thing is explained in the units section below.

File: content/docs/practices/naming.md (hunk 1 above)
Comment (beorn7) [inline content/docs/practices/naming.md]: Trying now...

File: content/docs/practices/naming.md (hunk 3 above)
Comment (beorn7) [inline content/docs/practices/naming.md]: Oops...

File: content/docs/practices/naming.md (hunk 1 above)
Comment (beorn7) [inline content/docs/practices/naming.md]: Yeah, worked.

File: content/docs/practices/naming.md (hunk 2 above)
Comment (juliusv) [inline content/docs/practices/naming.md]: This section says, "should have a suffix describing the unit", and then lists as one of the examples a metric without any unit, but a `_total` suffix. That's the confusing bit. It was there before, but we could make it clearer what suffix is expected when.

Comment (beorn7) [inline content/docs/practices/naming.md]: I see, I missed what line you are commenting on.

Will improve wording.

**Pull request merged**